build.gradle

```diff
@@ -1,112 +1,112 @@
-def kotlinVersion = '1.3.61'
-
-dependencies {
-    // Kotlin
-    compile "org.jetbrains.kotlinx:kotlinx-coroutines-core:${coroutinesVersion}"
-
-    // Grpc and Protobuf
-    compile 'com.google.protobuf:protobuf-java:3.11.0'
-    compile "io.grpc:grpc-netty-shaded:${grpcVersion}"
-    compile "io.grpc:grpc-protobuf:${grpcVersion}"
-    compile "io.grpc:grpc-stub:${grpcVersion}"
-
-    // Java
-    compile "javax.annotation:javax.annotation-api:1.2"
+subprojects {
+    plugins.withId("org.jetbrains.dokka") {
+        dokka {
+            outputFormat = 'html'
+            outputDirectory = "$buildDir/dokka"
+            configuration {
+                externalDocumentationLink {
+                    url = new URL("http://grpc.github.io/grpc-java/javadoc/")
+                }
+                externalDocumentationLink {
+                    url = new URL("http://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-core/")
+                }
+                perPackageOption {
+                    prefix = "io.grpc.testing"
+                    suppress = true
+                }
+                perPackageOption {
+                    prefix = "io.grpc.kotlin.generator"
+                    suppress = true
+                }
+            }
+        }
+    }
 
-    // Misc
-    compile 'com.squareup:kotlinpoet:1.5.0'
-    compile "org.jetbrains.kotlin:kotlin-reflect:1.3.61"
-    compile "com.google.truth:truth:1.0.1"
+    plugins.withId("maven-publish") {
+        publishing {
+            publications {
+                maven(MavenPublication) {
+                    pom {
+                        name = "grpc.io:" + project.name
+                        url = 'https://github.com/grpc/grpc-kotlin'
+                        afterEvaluate {
+                            // description is not available until evaluated.
+                            description = project.description
+                        }
+
+                        scm {
+                            connection = 'scm:git:https://github.com/grpc/grpc-kotlin.git'
+                            developerConnection = 'scm:git:git@github.com:grpc/grpc-kotlin.git'
+                            url = 'https://github.com/grpc/grpc-kotlin'
+                        }
+
+                        licenses {
+                            license {
+                                name = 'Apache 2.0'
+                                url = 'https://opensource.org/licenses/Apache-2.0'
+                            }
+                        }
+
+                        developers {
+                            developer {
+                                id = "grpc.io"
+                                name = "gRPC Contributors"
+                                email = "grpc-io@googlegroups.com"
+                                url = "https://grpc.io/"
+                                organization = "gRPC Authors"
+                                organizationUrl = "https://www.google.com"
```

Comment:
Organization URL should probably be https://www.linuxfoundation.org, right.
@lucperkins, @zacharysarah - WDYT?

Reply:
I agree with you, but I took this from `grpc-java`

https://github.com/grpc/grpc-java/blob/master/build.gradle#L379

So I think we should be consistent

Reply:
Hmm, I think that both should probably be updated, but I'll let @lucperkins et all chime in.